com.zettlr.Zettlr.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: com.zettlr.Zettlr
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+rename-icon: Zettlr
+command: start-zettlr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=xdg-documents
+  - --filesystem=/tmp
+  - --device=dri
+  - --env=PATH=/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin # required to find texlive binaries
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    version: '21.08'
+    directory: texlive
+    no-autodownload: true
+
+modules:
+  - name: texlive-extension
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/texlive
+
+  - name: zettlr
+    buildsystem: simple
+    build-commands:
+      - chmod +x Zettlr-2.1.2-*.AppImage
+      - ./Zettlr-2.1.2-*.AppImage --appimage-extract  
+      - mkdir -p "${FLATPAK_DEST}/zettlr"
+      - cp -r squashfs-root/* "${FLATPAK_DEST}/zettlr"
+      - mv "${FLATPAK_DEST}/zettlr/usr/share/icons" "${FLATPAK_DEST}/share"
+      - install -Dm644 "${FLATPAK_DEST}/zettlr/Zettlr.desktop" "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - desktop-file-edit --set-key=Exec --set-value="start-zettlr" "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - install -Dm 755 start-zettlr.sh "${FLATPAK_DEST}/bin/start-zettlr"
+      - rm -rf "${FLATPAK_DEST}/share/icons/hicolor/1024x1024/" # a 1024 is too large
+    post-install:
+      - install -D  com.zettlr.Zettlr.metainfo.xml -t "${FLATPAK_DEST}/share/metainfo/"
+    sources:
+      - type: file
+        url: https://github.com/Zettlr/Zettlr/releases/download/v2.1.2/Zettlr-2.1.2-x86_64.AppImage
```

Comment:
Should use https://github.com/flathub/flatpak-external-data-checker/ for automatic updates.

Reply:
Will do

Reply:
Hey @Lunarequest, any luck adding this? It's good to be merged apart from this.

Reply:
Sorry. I've been dealing with some legal stuff for the past few weeks along with uni so i haven't had a chance to touch this

Reply:
No worries, I've just been reviewing older submissions. Take your time (and take care!)